src/js/routes/TwitterHandleLanding.jsx

```diff
@@ -103,43 +103,43 @@ export default class TwitterHandleLanding extends Component {
     }
 
     const {
-      voter, kind_of_owner, owner_we_vote_id, twitter_handle,
+      voter, kindOfOwner, ownerWeVoteId, twitterHandle,
     } = this.state;
-    const signed_in_twitter = voter === undefined ? false : voter.signed_in_twitter;
-    let signed_in_with_this_twitter_account = false;
-    let looking_at_positions_for_friends_only = false;
-    if (signed_in_twitter) {
-      const twitter_handle_being_viewed = twitter_handle; // Variable copied for code clarity
+    const signedInTwitter = voter === undefined ? false : voter.signedInTwitter;
```

Comment:
The voter object is coming from the API server, and the API server uses variables with lower case with underscores. So this line should be:
`const signedInTwitter = voter === undefined ? false : voter.signed_in_twitter;`